custom-words.txt

```diff
@@ -1606,3 +1606,44 @@ Zilla
 ziplist
 Zoho
 zset
+userdelegationkey
+batchresponse
+policyid
+ruleid
+legalhold
+snaptshot
```

Comment:
This and a few others being added are typos, rather than words that should be globally accepted. File level suppressions can be done in `cspell.json` for temporary ignores till they can be properly fixed in the spec files